docs/docs/command-line-flags.md

```diff
@@ -276,15 +276,32 @@ The Web Site flags control the behavior of Marathon's web site, including the us
 *  <span class="label label-default">v0.10.0</span> `--http_max_concurrent_requests` (Optional.): the maximum number of
     concurrent http requests, that is allowed concurrently before requests get answered directly with a
     HTTP 503 Service Temporarily Unavailable.
+    
+### Metrics Flags
+
+* <span class="label label-default">v0.13.0</span> `--[disable_]metrics` (Optional. Default: enabled):
+    Expose the execution time per method via the metrics endpoint.
+    This metrics measurement can be disabled with --disable_metrics.
+* <span class="label label-default">v0.13.0</span> `--reporter_graphite` (Optional. Default: disabled):
+    Report metrics to [Graphite](http://graphite.wikidot.com) as defined by the given url. Example: tcp://localhost:2003?prefix=marathon-test&interval=10 
```

Comment:
It would be good to include an example with proper URL quoting to show that it might be necessary.